src/components/Aframe/my-loader.js

```diff
@@ -298,130 +298,118 @@ AFRAME.registerComponent("myloader", {
     }
   },
 
-  loadModel: function () {
-    var currentVolume = this.el.getObject3D("mesh");
-    if (currentVolume !== undefined) {
-      //clear mesh
-      currentVolume.geometry.dispose();
-      currentVolume.material.dispose();
-      this.el.removeObject3D("mesh");
-      this.el.sceneEl.object3D.dispose();
-      currentVolume = undefined;
-    }
+  loadModel: function (fullPath, volumeProperties) {
+
+	  var currentVolume = this.el.getObject3D('mesh');
+	  if (currentVolume !== undefined) {
+			//clear mesh
+		  currentVolume.geometry.dispose();
+		  currentVolume.material.dispose();
+		  this.el.removeObject3D('mesh');
+		  this.el.sceneEl.object3D.dispose();
+		  currentVolume = undefined;
+		}
+
+	   if (fullPath !== "") {
+		  this.hiddenLabel.style.display = '';
+		  var el = this.el;
+		  var data = this.data;		
+		  var myWidth = this.myCanvas.width;
```

Comment:
why the `my` prefixes instead of just `width` or `height`? are there other width/heights to distinguish these from?

Reply:
Just old notation before major changes. 
I'll change it.